examples/iris/main.py

```diff
@@ -32,6 +32,7 @@ def main():
             pf.Number(iris_df.shape[0], "Sample size"),
             pf.Number(len(iris_df.species.unique()), "Unique species"),
             pf.Number(iris_df.sepal_length.max(), "Max Sepal Length"),
+            pf.Kpi(482.537, 595.092, "Compare KPI"),
```

Comment:
For this example, I think it would be nicer to add 2 KPIs on a separate line. Perhaps for the taxi dataset for instance:
```python3
[
     pf.Kpi(round(taxi_df["tip"].mean(), 2), 3, "Average Tip"),
     pf.Kpi(taxi_df[taxi_df["tip"] < 1].shape[0], 1, "Tips under 1$"),
]
```